src/Volume.ts

```diff
@@ -200,6 +200,18 @@ export default class Volume {
     this.volumeDataObservers = [];
   }
 
+  private setUnloaded() {
+    this.loaded = false;
+    // TODO this will cause problems once it is possible to skip loading some channels. Come back to this then.
+    this.channels.forEach((channel) => {
+      channel.loaded = false;
```

Comment:
do you want to empty out the channel's data arrays here too?

Reply:
I think those can stay, rather than risking trying to redraw an empty channel while new data is being loaded for it.

The problem this function solves btw was with functions that trigger loading and receive a per-channel callback. It's common for those callbacks to call `Volume.isLoaded` to ensure that some code runs only once, after every channel has loaded. But if `Volume` had already been loaded into at least once, `Volume.isLoaded()` was coming back true on every callback, not just the last one.